gammapy/estimators/points/core.py

```diff
@@ -740,3 +788,127 @@ def recompute_ul(self, n_sigma_ul=2, **kwargs):
             )
         flux_points.meta["n_sigma_ul"] = n_sigma_ul
         return flux_points
+
+    def get_rebinned_axis(self, method, value, axis_name="energy"):
```

Comment:
I think for now I would male this an independent helper method. We have the same for datasets: https://github.com/gammapy/gammapy/blob/main/gammapy/estimators/utils.py#L254